ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -226,11 +227,16 @@ def start_span(
         # Use the specified parent or the active span if possible. Otherwise,
         # use a `None` parent, which triggers the creation of a new trace.
         parent = child_of.unwrap() if child_of else None
-        span = self._otel_tracer.create_span(operation_name, parent)
 
+        links = None
```

Comment:
```suggestion
        links = ()
```

An empty tuple should be a singleton constant, so no performance overhead over using `None` here.